proto/c1/connector/v2/resource.proto

```
@@ -76,9 +76,13 @@
~
~service ResourceManagerService {
~  rpc CreateResource(CreateResourceRequest) returns (CreateResourceResponse);
   rpc DeleteResource(DeleteResourceRequest) returns (DeleteResourceResponse);
 }
 
+service ResourceDeleterService {
+  rpc DeleteResource(DeleteResourceRequest) returns (DeleteResourceResponse);
```

Buf (1.47.2)

"c1.connector.v2.DeleteResourceRequest" is used as the request or response type for multiple RPCs.

(RPC_REQUEST_RESPONSE_UNIQUE)